packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverBody/TeachingPopoverBody.types.ts

```diff
@@ -1,17 +1,21 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type TeachingPopoverBodyMediaLength = 'short' | 'medium' | 'tall';
 
 export type TeachingPopoverBodySlots = {
-  root: Slot<'div'>;
+  /**
+   * The element wrapping the buttons.
+   */
+  root: NonNullable<Slot<'div'>>;
+  /**
+   * Optional Media Content.
+   */
+  media?: Slot<'span'>;
 };
 
-/**
- * TeachingPopoverBody Props
- */
-export type TeachingPopoverBodyProps = ComponentProps<TeachingPopoverBodySlots> & {};
+export type TeachingPopoverBodyProps = ComponentProps<Partial<TeachingPopoverBodySlots>> & {
```

Comment:
```suggestion
export type TeachingPopoverBodyProps = ComponentProps<TeachingPopoverBodySlots> & {
```

I don't think that `Partial` is needed there